src/tests/test_logged_activities.py

```diff
@@ -516,6 +516,209 @@ def test_secretary_edit_logged_activity_empty_payload(self):
         self.assertEqual(response.status_code, 400)
 
 
+class LoggedActivityApprovalTestCase(BaseTestCase):
+    """Logged activities approval tests"""
+
+    def test_approving_logged_activities_successful(self):
+
```

Comment:
`pep8`

Reply:
Explain please.

Reply:
The formatting is off. Look at the link I sent you, they have an example on how to do it.

Reply:
I hope I've fixed this. Check it out. Thanks.